src/Tempest/Framework/Testing/Http/TestResponseHelper.php

```diff
@@ -177,9 +180,35 @@ public function assertHasNoValidationsErrors(): self
             sprintf(
                 "There should be no validation errors, but there were: %s",
                 implode(', ', array_keys($validationErrors)),
-            )
+            ),
```

Comment:
did you add this purely because of the possible `message` argument?

Reply:
This was my autoformatter, I don't know why php cs fixer didn't pick up on it though